internal/dockerutil/setup.go

```diff
@@ -145,21 +123,50 @@ func dockerCleanup(t DockerSetupTestingT, cli *client.Client) func() {
 					b := new(bytes.Buffer)
 					_, err := b.ReadFrom(rc)
 					if err == nil {
-						t.Logf("Container logs - {%s}\n%s", strings.Join(c.Names, " "), b.String())
+						t.Logf("\n\nContainer logs - {%s}\n%s", strings.Join(c.Names, " "), b.String())
```

Comment:
why do we need the two newlines at the beginning of this log now?

Reply:
This is purely for readability. Without this, the log dump between each node bud up against each other and it is difficult to see where the logs from one node end and the other begins.